tests/nested/manual/core20-early-config/task.yaml

```diff
@@ -99,3 +109,19 @@ execute: |
     remote.exec "sudo snap wait system seed.loaded"
 
     check_core20_early_config
+
+    # XXX: this should be a different test maybe?
```

Comment:
maybe, though new nested tests have their cost

Reply:
Thanks, removed it for now